mappings/net/minecraft/entity/boss/BossBar.mapping

```diff
@@ -21,7 +21,7 @@ CLASS net/minecraft/class_1259 net/minecraft/entity/boss/BossBar
 		ARG 1 style
 	METHOD method_5410 setDragonMusic (Z)Lnet/minecraft/class_1259;
 		ARG 1 dragonMusic
-	METHOD method_5411 setThickenFog (Z)Lnet/minecraft/class_1259;
+	METHOD method_5411 shouldThickenFog (Z)Lnet/minecraft/class_1259;
 		ARG 1 thickenFog
```

Comment:
Is this really a getter?

Reply:
Fixed in f31d2fe249ebc0d2e451b1f569841dcd97077e54